### PR TITLE
Matrix4: Changed .makeFrustum() to .makePerspective()

### PR DESCRIPTION
**Pull request by Mugen87**
see #10265

### CODE REVIEW

File: src/math/Matrix4.js [hunk 1]
```diff
@@ -833,7 +833,7 @@ Matrix4.prototype = {
 
 	}(),
 
-	makeFrustum: function ( left, right, bottom, top, near, far ) {
+	makePerspective: function ( left, right, bottom, top, near, far ) {
```
Comment (mrdoob) [inline src/math/Matrix4.js]: You don't want to change this to `left, right, top, bottom, near, far` so it matches `makeOrthographic`?

Comment (Mugen87) [inline src/math/Matrix4.js]: I was unsure about that. On the other hand, this is the right moment to correct this inconsistency. I'll change the code.

**Comment by mrdoob** [comment]
@WestLangley looks good?

**Comment by WestLangley** [comment]
Looks good to me.

Can we add a warning if `MakePerspective()` is called with `far` undefined? This would indicate that the user is assuming the old 4-arg method. Something like:

    Warning: MakePerspective() has been redefined and has a new signature. Please check the docs.

**Comment by mrdoob** [comment]
Thanks guys!

**Comment by bhouston** [comment]
BTW: 

(1) Matrix4makePerspective was intended to be an implementaiton of venerable gluPerspective function from OpenGL here: https://www.opengl.org/sdk/docs/man2/xhtml/gluPerspective.xml

(2) Matrix4.makeFrustum was intended to be an implementaiton of the standard OpenGL function glFrustum: https://www.opengl.org/sdk/docs/man2/xhtml/glFrustum.xml

I am okay with removing makePerspective but you shouldn't then rename makeFrustum to makePerspective and change makePerspective's signature.

You are just messing with non-standard terminology and also breaking a bunch of code - like mine.  I am getting warnings for what is clearly incorrect behavior.  I would have preferred that makePerspective just was completely removed rather than essentially breaking it.

I would obsolete the old makePerspective if you feel strongly about it, but this renaming is confusing as we were following OpenGL terminology and now we are not.

**Comment by WestLangley** [comment]
See continued discussion in #10265.

**Pull request merged**